includes/class-wc-ecster-templates.php

```diff
@@ -118,5 +121,47 @@ public function add_customer_type_switch() {
 			<?php
 		}
 	}
+
+	/**
+	 * Add checkout page body class, depending on checkout page layout settings.
+	 *
+	 * @param array $class CSS classes used in body tag.
+	 * @return array The same input array with the addition of our custom classes.
+	 */
+	public function add_body_class( $class ) {
+		if ( ! is_checkout() || is_wc_endpoint_url( 'order-received' ) ) {
+			return $class;
+		}
+
+		if ( method_exists( WC()->cart, 'needs_payment' ) && ! WC()->cart->needs_payment() ) {
+			return $class;
+		}
+
+		$settings        = get_option( 'woocommerce_ecster_settings' );
+		$checkout_layout = $settings['checkout_layout'] ?? 'two_column_right';
```

Comment:
@montazarr Does this mean that a merchant updating the plugin to this version will get a two column checkout if the don't save the settings?